packages/toolkit/connector-kit/src/types.ts

```diff
@@ -183,13 +183,21 @@ export type GetSession = () => Promise<ConnectorSession>;
 
 export type SetSession = (storage: ConnectorSession) => Promise<void>;
 
-export const storageGuard = z.record(unknown());
+export const storageGuard = z
+  .object({
+    accessToken: z.string(),
+    refreshToken: z.string(),
+    expiresIn: z.number(),
+    tokenType: z.string(),
+  })
+  .partial()
+  .catchall(z.union([z.string(), z.number(), z.boolean(), z.null()]));
```

Comment:
why do we need `catchall` here?

Reply:
Previously was considering use can compute the expiring time and choose whether to exchange a new AT using the refresh token. Remove `catchall` to stick to the standard.